### ISSUE TITLE
[São Paulo/SP] - Cientista de Dados

### ISSUE THREAD

**Issue by rodrigoleite89** [tensorflow-brasil/vagas#1]
## Descrição da vaga

Projeto de reconhecimento facial com TensorFlow, etc...

## Local

Remoto ou Escritório, São Paulo - Vila Madalena

## Benefícios

- Plano de saúde
- Seguro de vida
- VR de R$ X/dia
- Auxílio creche
- ...

#### Diferenciais

- 2 horas de almoço
- monte a sua máquina como desejar
- frigobar sempre cheio :P
- sala de jogos
- ...

## Requisitos

- TensorFlow, Scikit-Learn, Matplotlib, NumPy

**Obrigatórios:**
- Experiência com Sklearn e TensorFlow
- 1 ano de experiência com desenvolvimento de aplicações com Machine Learning

**Desejáveis:**
- Conhecimentos em programação paralela em GPU usando CUDA

**Diferenciais:**
- projetos opensource
- Ter trabalhado com Machine Learning e/ou Big Data por pelo menos 2 anos

## Contratação

PJ a combinar

## Nossa empresa

Aqui na Corporação TensorFlow Brasil somos líderes em...

## Como se candidatar

Por favor envie um email para email@email.com.br com seu CV anexado - enviar no assunto: Vaga TensorFLow

## Labels

- alocação/Flexível
- alocação/Presencial
- alocação/Remoto
- experiência/Estágio
- experiência/Júnior
- experiência/Pleno
- experiência/Sênior
- regime/A-Combinar
- regime/CLT
- regime/Estágio
- regime/Freela
- regime/Outros
- regime/PJ
- remuneração/1k-3k
- remuneração/3k-5k
- remuneração/5k-10k
- remuneração/10k-15k
- remuneração/15k+
